config.mak.uname

```diff
@@ -335,6 +335,16 @@ endif
 ifeq ($(uname_S),Windows)
 	GIT_VERSION := $(GIT_VERSION).MSVC
 	pathsep = ;
+	# Prepend MSVC 64-bit tool-chain to PATH.
+	#
+	# TODO A normal git-bash window DOES NOT contain the path to CL.EXE
+	# TODO and friends in $PATH.
```

Comment:
I think we can probably auto-detect the latest `vcvars.bat` using a wildcard expression. Will try to get that to work later this week.